### DISCUSSION TITLE
5G Roaming Blog Post

### DISCUSSION THREAD

**Discussion by infinitydon** [HewlettPackard/PacketRusher#53] [Show and tell]
Hi,

I just released a blog post about using Packet-Rusher and Open5gs to deploy 5G roaming, link below:

https://futuredon.medium.com/5g-roaming-with-mutual-tls-1468d109129c

**Comment by linouxis9** [comment]
Hi @infinitydon,

Thanks a lot for this blog post! It is a great demonstration of Open5GS and PacketRusher's capabilities, and I learned a lot about 5G Roaming with it.
I've re-shared your blog post on LinkedIn :-)

Cheers!
Valentin

**Reply by infinitydon** [reply]
Thanks
